build_ngtf.py

```diff
@@ -158,7 +158,8 @@ def main():
 
     if arguments.use_prebuilt_tensorflow:
         print("Using existing TensorFlow")
-        command_executor(["pip", "install", "-U", "tensorflow==" + tf_version])
+        #command_executor(["pip", "install", "-U", "tensorflow==" + tf_version])
+        command_executor(["pip", "install", "-U", "tf-nightly"])
```

Comment:
Otherwise we cannot install pre-built TensorFlow (as there is no tag on 1.14 yet). Once we do - will replace this with the tag (e.g., `v1.14.0-rc0` or something.